compiler/passes/InitNormalize.cpp

```diff
@@ -905,8 +905,9 @@ void InitNormalize::fieldInitTypeInference(Expr*    insertBefore,
 ************************************** | *************************************/
 
 bool InitNormalize::isFieldAccessible(Expr* expr) const {
-  AggregateType* at     = type();
-  bool           retval = true;
+  AggregateType* at      = type();
+  bool           initNew = hasInitDone(mFn->body);
```

Comment:
Not specific to this PR, but could we store this 'initNew' property in the ``InitNormalize`` class?